libs/shared/cms/src/lib/queries/iap-offerings-by-storeids/util.ts

```diff
@@ -49,15 +50,25 @@ export class IapOfferingsByStoreIDsResultUtil {
     return false;
   }
 
-  getProductNamesByStoreIds(
+  getIapPageContentByStoreIds(
     storeIds: string[]
-  ): Record<string, string | undefined> {
-    const result: Record<string, string | undefined> = {};
+  ): Record<string, { productName: string; supportUrl: string }> {
+    const result: Record<string, { productName: string; supportUrl: string }> =
+      {};
+
     for (const storeId of storeIds) {
       const iap = this.iaps.find((iap) => iap.storeID === storeId);
-      result[storeId] =
-        iap?.offering?.defaultPurchase?.purchaseDetails?.productName;
+      if (!iap) throw new IapPageContentNotFoundForStoreIdError(storeId);
+
+      const productName =
+        iap.offering.defaultPurchase.purchaseDetails.productName;
+      const supportUrl =
+        iap.offering.commonContent.localizations[0].supportUrl ??
+        iap.offering.commonContent.supportUrl;
```

Comment:
suggestion: Return fields and localizations separately

The pattern followed by other utilities is to return the fields and localization separately and let the consumers determine how to resolve the correct value.

Reply:
Also, is `supportUrl` actually localize-able? I believe `productName` is.